<launch>
    <!--set runtime platform kernel parameters-->
    <param name="publish_robot_id_duration" type="double" value="0.1"/>
    <param name="publish_swarm_list_duration" type="double" value="5.0"/>
    <param name="default_neighbor_distance" type="double" value="50"/>
    <param name="total_robot_numbers" type="int" value="20"/>

    <group ns="robot_0">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node0" output="screen">
            <param name="unique_robot_id" type="int" value="0"/>
        </node>
    </group>
    <group ns="robot_1">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node1" output="screen">
            <param name="unique_robot_id" type="int" value="1"/>
        </node>
    </group>
    <group ns="robot_2">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node2" output="screen">
            <param name="unique_robot_id" type="int" value="2"/>
        </node>
    </group>
    <group ns="robot_3">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node3" output="screen">
            <param name="unique_robot_id" type="int" value="3"/>
        </node>
    </group>
    <group ns="robot_4">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node4" output="screen">
            <param name="unique_robot_id" type="int" value="4"/>
        </node>
    </group>
    
    <group ns="robot_5">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node5" output="screen">
            <param name="unique_robot_id" type="int" value="5"/>
        </node>
    </group>
    <group ns="robot_6">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node6" output="screen">
            <param name="unique_robot_id" type="int" value="6"/>
        </node>
    </group>
    <group ns="robot_7">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node7" output="screen">
            <param name="unique_robot_id" type="int" value="7"/>
        </node>
    </group>
    <group ns="robot_8">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node8" output="screen">
            <param name="unique_robot_id" type="int" value="8"/>
        </node>
    </group>
    <group ns="robot_9">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node9" output="screen">
            <param name="unique_robot_id" type="int" value="9"/>
        </node>
    </group>

    <group ns="robot_10">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node10" output="screen">
            <param name="unique_robot_id" type="int" value="10"/>
        </node>
    </group>
    <group ns="robot_11">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node11" output="screen">
            <param name="unique_robot_id" type="int" value="11"/>
        </node>
    </group>
    <group ns="robot_12">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node12" output="screen">
            <param name="unique_robot_id" type="int" value="12"/>
        </node>
    </group>
    <group ns="robot_13">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node13" output="screen">
            <param name="unique_robot_id" type="int" value="13"/>
        </node>
    </group>
    <group ns="robot_14">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node14" output="screen">
            <param name="unique_robot_id" type="int" value="14"/>
        </node>
    </group>

    <group ns="robot_15">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node15" output="screen">
            <param name="unique_robot_id" type="int" value="15"/>
        </node>
    </group>
    <group ns="robot_16">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node16" output="screen">
            <param name="unique_robot_id" type="int" value="16"/>
        </node>
    </group>
    <group ns="robot_17">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node17" output="screen">
            <param name="unique_robot_id" type="int" value="17"/>
        </node>
    </group>
    <group ns="robot_18">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node18" output="screen">
            <param name="unique_robot_id" type="int" value="18"/>
        </node>
    </group>
    <group ns="robot_19">
        <node pkg="micros_swarm_framework" type="rtp_manager_node" name="rtp_manager_node19" output="screen">
            <param name="unique_robot_id" type="int" value="19"/>
        </node>
    </group>
</launch>

